frontend/src/stores/recorder.ts

```diff
@@ -3,14 +3,18 @@ import { ApiPaths, ApiResponse } from "../types/Api";
 import { Notify } from "quasar";
 import axios from "axios";
 import { Recorder } from "../types/Recorder";
+import { date } from 'quasar'
 
 const axiosInstance = axios.create();
 axiosInstance.defaults.timeout = 600;
 
+const timeStamp = Date.now()
+const formattedTimeStamp = date.formatDate(timeStamp, 'YYYYMMDD_HHmmss')
```

Comment:
Can you extract `formattedTimeStamp` logic to some helper file so implementation is not repeated across different files?